dtmi/fii/fii/badge_apt-1.json

```diff
@@ -0,0 +1,38 @@
+{
+    "@id": "dtmi:fii:fii:badge_apt;1",
+    "@type": "Interface",
+    "contents": [
+        {
+            "@type": "Telemetry",
+            "displayName": {
+                "en": "Gsensor"

```

Comment:
Consider expanding the displayName or adding a description which expands on the meaning of "Gsensor".

Reply:
Hi @digimaun 
I have update model file to remove Gsensor part first.